CHANGELOG.md

```diff
@@ -39,10 +39,10 @@ Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
 + Add - Readers for: `SpikeGLX`, `Open Ephys`, `Kilosort`
 + Add - Probe table supporting: Neuropixels probes 1.0 - 3A, 1.0 - 3B, 2.0 - SS, 2.0 - MS
 
-[0.1.4]: https://github.com/datajoint/element-array-ephys/releases/tag/0.1.4
-[0.1.3]: https://github.com/datajoint/element-array-ephys/releases/tag/0.1.3
-[0.1.2]: https://github.com/datajoint/element-array-ephys/releases/tag/0.1.2
-[0.1.1]: https://github.com/datajoint/element-array-ephys/releases/tag/0.1.1
+[0.1.4]: https://github.com/datajoint/element-array-ephys/compare/0.1.3...0.1.4
```

Comment:
@kabilar We recently merged on workflow-ephys/caimg in favor of tag links, not compare links. Should we adopt one strategy over the other across all u24 repos?